src/apis/index.js

```diff
@@ -191,6 +192,7 @@ export default {
   setLayoutMode,
   setMaxZoomLevel,
   setMinZoomLevel,
+  setMomentumScrollEnabled, // Diligent
```

Comment:
nitpick so feel free to ignore but all the other webviewer-ui configs use a naming scheme like `enableMomentumScroll` (e.g. see enableDownload.js - it still takes a bool for whether to enable or disable.